### ISSUE TITLE
Can this run standalone as an altering aggregator without Hawkular?

### ISSUE THREAD

**Issue by darrenhaken** [hawkular/hawkular-alerts#497]

**Comment by lucasponce** [comment]
Yes, technically you can run Hawkular Alerting as a java process and with version 2 you wouldnt need any additional component from Hawkular suite (metrics or other).

**Comment by darrenhaken** [comment]
I was interested in using it to centralise/federate alerts from Prometheus and Elastic (without the alerting x-pack). Is this doable then?

Not sure if Elastic is achievable without the alerting module.

**Comment by lucasponce** [comment]
Yes, that was the goal of the v2 project, take a look into the 
http://www.hawkular.org/blog/2017/08/alerts-multiple-sources.html

It's a community project (meaning that is *not* intended to be part of any commercial offering) but if it suits your needs, feel free to use it or even contribute to it.

**Comment by darrenhaken** [comment]
@lucasponce - to be clear you are saying *it is* intended to be part of a commercial product?

I'll take a look and see how it works :)

**Comment by lucasponce** [comment]
@darrenhaken you are right :-), sorry, fast typing.
Just that it won't be part of any commercial product, just a community project for now.
